### ISSUE TITLE
Graph: Focus order is not logical after selecting 'Graph' tree item present under "Gremlin API" section of 'Data Explorer' page.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1059]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1247221

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Gremlin(graph)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate  to "New Graph" menu item and select it.
Navigate through all the controls under Add Graph blade and enter details and selct ok button to create new 'Graph'.
Create 'Sample Graph under 'Gremlin  API' section and expand created graph control. 
Navigate through all the controls under "Gremlin API" Section and select "Graph" which appears after expending the items.
Press tab key after selecting Graph tree item.
Verify the issue.

Actual Result:
Focus order is not logical after selecting 'Graph' tree item present under "Gremlin API" section of 'Data Explorer' page. After selecting 'Graph' option focus moves to rest of the tree items and expand/collapse controls present and then to 'Graph' tab control which is not logical.

Expected Result:
Focus order should be logical after selecting 'Graph' tree item present under "Gremlin API" section of 'Data Explorer' page. After selecting 'Graph' option and pressing tab key focus should move to 'Graph' tab control.'


Note:
This issue exists throughout the application.
'Tabstop' of 'Accessibility Insights for Web' tool is not working on this page.



https://user-images.githubusercontent.com/8519407/132712887-b2ce14a6-9a28-41ad-9ded-0170dc089746.mp4
